### ISSUE TITLE
Postgres database error in V3

### ISSUE THREAD

**Issue by Phu1237** [nuxt/content#2884]
### Environment

Nuxt project info:                                                                                                                                                     10:11:37 PM

------------------------------
- Operating System: Windows_NT
- Node Version:     v22.11.0
- Nuxt Version:     3.14.1592
- CLI Version:      3.16.0
- Nitro Version:    2.10.4
- Package Manager:  bun@1.1.38
- Builder:          -
- User Config:      default
- Runtime Modules:  @nuxt/ui-pro@3.0.0-alpha.9, @nuxtjs/i18n@9.0.0-rc.2, @nuxt/content@3.0.0-alpha.7
- Build Modules:    -
------------------------------

`compatibilityVersion` is set to 4

### Reproduction

My content config follows as a V3 document https://content3.nuxt.dev/docs/getting-started/configuration#postgres
```
  content: {
    database: {
      type: "postgresql",
      url: process.env.POSTGRES_URL ?? "http://localhost",
      /* Other options for `pg` */
    },
  },
```

Run
```bash
nuxi build 
```

The console output this error
> Could not load .../node_modules/@nuxt/content/dist/runtime/adapters/postgresql (imported by node_modules/@nuxt/content/dist/runtime/internal/database.server.js): ENOENT: no such file or directory, open '...\node_modules\@nuxt\content\dist\runtime\adapters\postgresql'

### Describe the bug

I want to use PostgreSQL as content database but always meet error when build

I think it is missing postgres code somewhere because when I change the type to `postgres` to match with the adapter name
![Image](https://github.com/user-attachments/assets/d82bb77d-9743-4ec4-b0ae-b23278a3323f)
It shows me this
![Image](https://github.com/user-attachments/assets/d8d0b375-245c-45d0-9203-d4a38fa1714a)

Please help me to check it
Thank you!

### Logs

```Shell

```

**Comment by farnabaz** [comment]
Seems that the types are wrong, should be fixed in `https://pkg.pr.new/@nuxt/content@36d3b08` and will release in `v3.0.0-alpha.8`

**Comment by Phu1237** [comment]
@farnabaz 
Thank you for your hard work!
I will check it and give feedback when the new alpha version released
Thanks!

**Comment by farnabaz** [comment]
Released in [v3.0.0-alpha.8](https://github.com/nuxt/content/releases/tag/v3.0.0-alpha.8).
